### PR TITLE
chore: add volunteer bobbyy16

### PR DESCRIPTION
**Pull request by bobbyy16**

### CODE REVIEW

**Comment by JasonDsouza212** [comment]
@bobbyy16  You are **not allowed** to give your own slug and topics. Check the allowed slug and add accordingly.

**Comment by bobbyy16** [comment]
Done @JasonDsouza212

File: data/users/bobbyy16.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Abhishek Kumara",
+  "bio": "DevOps engineer",
+  "image": "https://avatars.githubusercontent.com/u/62874675?s=400&u=8cef03bf888fe01b1ef247da5a714e8629442ade&v=4",
```
Comment (codewithvoid) [inline data/users/bobbyy16.json:4]: use simpler url here https://avatars.githubusercontent.com/u/62874675?v=4

**Comment by bobbyy16** [comment]
changed to simpler url @codewithvoid

**Comment by codewithvoid** [comment]
Perfect. Welcome!

**Comment by bobbyy16** [comment]
thank you

**Pull request merged**